config/initializers/secure_headers.rb

```diff
@@ -10,6 +10,6 @@
     :connect_src => ["'self'", 'ws:', 'wss:'],
```

Comment:
On a related note: probably fine to drop `ws:` here if SSL is used and required.

Reply:
I think this is only used for webpack dev-server currently, but may be wrong. anyways outside of the scope here.

Reply:
Hence the approval. I think then the tests were still running so I didn't merge.